app/.env.sample

```diff
@@ -5,3 +5,5 @@ NF_PIPELINE_URL=https://github.com/jonk-ebi/platform-test-pipeline
 NF_WORK_DIR=/path/to/work/dir
 SEQERA_API=https://api.cloud.seqera.io
 UPLOAD_DIRECTORY=/tmpdir
+WEB_METADATA_API=https://beta.ensembl.org/api/metadata/
+VEP_SUPPORT_PATH=/nfs/public/ro/enswbsites/dev/vep/support/
```

Comment:
Why do we call it a "support" path? This name doesn't really tell much. VEP data files path? Or, if this is only for genomic data, then perhaps VEP genebuild path or VEP assembly path?

Reply:
Based on the code review discussion. This is good enough for VEP phase 1. Will need to revisit this once we have different files and how production is going to save it and how web is going to access it.